lib/binaryParsers.js

```diff
@@ -114,7 +114,7 @@ var parseFloat64 = function(value) {
 var parseNumeric = function(value) {
```

Comment:
If this parser is changing to return a string, I think it should return an exact string instead of being limited to double precision. It’s a bit misleading otherwise.

Reply:
Yes I was leaning toward that too. Best rationale I could come up with the numeric to string hack was the simplicity of maintaining the current representation as text. But if it's going to be fixed you're right that it should be fixed for good. If anybody else wants to take a stab at this feel free. Otherwise I'll circle back some time soon.

Reply:
(Working on it: https://github.com/charmander/pg-numeric/blob/master/index.js)

Reply:
I took a peek and looks very nice.

In particular the round trip with the COPY to generate the binary data for test vectors is genius!